src/client/components/Editor.tsx

```diff
@@ -32,6 +34,37 @@ class Editor extends React.PureComponent<EditorProps> {
         }
     }
 
+  componentDidMount(): void {
+      // TODO: why does using gutterclick yield weird highlighting?
+      this.aceComponent.current.editor.on("gutterclick", function(e) {
+        var target = e.domEvent.target;
+
+        if (target.className.indexOf("ace_gutter-cell") == -1){
+            return;
+        }
+
+        if (!e.editor.isFocused()){
+            return;
+        }
```

Comment:
Do you know why the editor needs to be focused for us to want to do this?

Reply:
I do not. It seems to work fine without this check, so I removed it.